Task3/Greedy Algorithm

```diff
@@ -0,0 +1,23 @@
+class Coin {
```

Comment:
Please give correct extension to code file (.java in this case)

Reply:
Ok.